python-package/lightgbm/basic.py

```diff
@@ -2261,9 +2261,11 @@ def predict(self, data, num_iteration=None,
 
             Note
             ----
-            If you want to get more explanation for your model's predictions using SHAP values
+            If you want to get more explanations for your model's predictions using SHAP values
             like SHAP interaction values,
-            you can install shap package (https://github.com/slundberg/shap).
+            you can install the shap package (https://github.com/slundberg/shap). Note that unlike
+            the shap package, with pred_contrib we return a matrix with an extra column, where the
```

Comment:
Ah, and one more please: emphasize `pred_contrib`   with ``, like other params, e.g. feature_name here:
https://github.com/Microsoft/LightGBM/blob/beb35d567de899b140bd61e174ef3b9ef5fd0769/python-package/lightgbm/basic.py#L688